### ISSUE TITLE
"Cannot find module" when using VSCodium

### ISSUE THREAD

**Issue by eratio08** [mattbierner/vscode-emojisense#49]
I wondered why the auto completion does not work, so I tried to trigger it via command pallet and got the following error:
<img width="323" alt="image" src="https://user-images.githubusercontent.com/5063643/148735088-e7d8a255-d18d-4432-a0dc-f30963b59df8.png">

Checking the extension host log I found:
```shell
[exthost] [error] Error: Cannot find module '/Users/user/.vscode-oss/extensions/bierner.emojisense-0.9.0/dist/extension'
Require stack:
- /Applications/VSCodium.app/Contents/Resources/app/out/vs/loader.js
- /Applications/VSCodium.app/Contents/Resources/app/out/bootstrap-amd.js
- /Applications/VSCodium.app/Contents/Resources/app/out/bootstrap-fork.js
	at Function.Module._resolveFilename (internal/modules/cjs/loader.js:887:15)
	at Module._load (internal/modules/cjs/loader.js:732:27)
	at Function.f._load (electron/js2c/asar_bundle.js:5:12913)
	at Function.t._load (/Applications/VSCodium.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:114:32222)
	at Function.p._load (/Applications/VSCodium.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:114:28887)
	at Function.f._load (/Applications/VSCodium.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:104:61699)
	at Module.require (internal/modules/cjs/loader.js:959:19)
	at require (internal/modules/cjs/helpers.js:88:18)
	at Function.r [as __$__nodeRequire] (/Applications/VSCodium.app/Contents/Resources/app/out/vs/loader.js:5:101)
	at v._loadCommonJSModule (/Applications/VSCodium.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:114:30297)
	at v._doActivateExtension (/Applications/VSCodium.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:100:12958)
	at v._activateExtension (/Applications/VSCodium.app/Contents/Resources/app/out/vs/workbench/services/extensions/node/extensionHostProcess.js:100:11931)
	at processTicksAndRejections (internal/process/task_queues.js:93:5)
	at async Promise.all (index 2)
	at async Promise.all (index 0)
```
I'm using 
:emojisense: v0.9.0
and VSCodium
Version: 1.63.2
Commit: 899d46d82c4c95423fb7e10e68eba52050e30ba3
Date: 2021-12-17T00:22:15.020Z (3 wks ago)
Electron: 13.5.2
Chromium: 91.0.4472.164
Node.js: 14.16.0
V8: 9.1.269.39-electron.0
OS: Darwin x64 21.2.0

Not sure if the is an issue of the plugin or vscodium. (Other plugins work as expected.)

**Comment by mattbierner** [comment]
How did you install this extension on vscodium?

**Comment by eratio08** [comment]
Via the extension view/page. Searched for it and clicked install. 
<img width="513" alt="image" src="https://user-images.githubusercontent.com/5063643/148908020-6e436f45-00f4-421d-8493-99c43abc903a.png">

Already tried clean re-install of the extension. Sadly did not change anything.

**Comment by mattbierner** [comment]
Thanks. To my knowledge the builds on vscodium are not official. I don't know what is wrong and don't have the time to support vscodium

I'd take a PR if someone wants to investigate and fix this but closing as I have not seen reports of this with the official build

**Comment by eratio08** [comment]
Manually download the extension and installing it with `codium --install-extension bierner.emojisense-0.9.0.vsix` did resolve it.

**Comment by Morikko** [comment]
I guess the VScodium pipeline use the last release (0.9.0) that does not contain the fix https://github.com/mattbierner/vscode-emojisense/commit/d9a4870d08b85dcd7d1ead28f0b84c652fdc734d . Creating a tag `0.9.1` would include the fix as the pipeline would automatically look for the latest new version.

Workaround for now: `mv ~/.vscode-oss/extensions/bierner.emojisense-0.9.0/build/dist/extension.js ~/.vscode-oss/extensions/bierner.emojisense-0.9.0/dist/` + restart the extension.

**Comment by Bhupesh-V** [comment]
> I guess the VScodium pipeline use the last release (0.9.0) that does not contain the fix [d9a4870](https://github.com/mattbierner/vscode-emojisense/commit/d9a4870d08b85dcd7d1ead28f0b84c652fdc734d) . Creating a tag `0.9.1` would include the fix as the pipeline would automatically look for the latest new version.
> 
> Workaround for now: `mv ~/.vscode-oss/extensions/bierner.emojisense-0.9.0/build/dist/extension.js ~/.vscode-oss/extensions/bierner.emojisense-0.9.0/dist/` + restart the extension.

Thanks for this, it worked 👍🏽
